app/frontend/src/services/formService.js

```diff
@@ -220,10 +230,15 @@ export default {
    * @function listSubmissions
    * Get the submissions for a form
    * @param {string} formId The form uuid
+   * @param {Boolean} deleted TRUE to show deleted submissions, FALSE to show only active ones
    * @returns {Promise} An axios response
    */
-  listSubmissions(formId) {
-    return appAxios().get(`${ApiRoutes.FORMS}/${formId}/submissions`);
+  listSubmissions(formId, deleted) {
```

Comment:
Setting deleted to be an optional but default to false I think would be better. Allows for "less" functional signature change, but offers more flexibility.
```js
listSubmissions(formId, deleted = false) { ... }
```